.github/workflows/lint.yaml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - loop
```

Comment:
No needed! At least in theory :) As you can see on the following line, the linter should run on git on every pull request.

If what you want is for the linter to run when you commit stuff, just do `pip install pre-commit`, then `pre-commit install`. This will install the pre-commit hook that runs whenever you try to commit any code.

Reply:
just added this because my pull request was failing some checks without it, but I can try pre-commit install